packages/ui/src/components/Visualization/Canvas/CanvasForm.tsx

```diff
@@ -122,7 +123,14 @@ export const CanvasForm: FunctionComponent<CanvasFormProps> = (props) => {
     return schema?.schema && schema.schema['$comment']?.includes('loadbalance');
   }, [schema]);
 
-  return schema?.schema === undefined ? null : (
+  return schema?.schema === undefined || Object.keys(schema?.schema).length === 0 ? (
+    <>
+      <Title headingLevel="h1">{componentName}</Title>
```

Comment:
[:nail_care:] Maybe it would be nice to extract this into a dedicated component.